README.md

```diff
@@ -15,8 +15,35 @@
 
 # <a name="overview"></a>Overview
 
-Each directory in this repository contains an automation workflow template for a specific community-built integration for Orbit workspaces.
+Inside this repository are template YAML workflow files you can use to automate **3rd party integrations** with your Orbit workspace.
 
+**What are 3rd party integrations?**
+
+All Orbit users have access to a host of [built-in integrations](https://orbit.love/integrations/), such as GitHub, Twitter, Slack, Discord and others. These built-in integrations can be added and set up from within the Orbit app.
+
+Community-built, or 3rd party integrations, are additional integrations offered by the larger Orbit community. While not officially supported, they offer many opportunities to further extend the utility of the platform. 
+
+**What is this repository for?**
+
+Within each subfolder of this repository is a GitHub Actions workflow template file, along with instructions on how to use it. GitHub Actions is a runtime environment provided by GitHub to any GitHub user. You can run applications on GitHub Actions for free on any publicly availably (*not private*) repository.
+
+**What integration templates are available here?**
+
+Currently, this repository contains integration templates for:
+
+* [DEV Community](https://github.com/orbit-love/github-actions-templates/blob/main/DEV)
+
+**How do I use these template files?**
+
+To use these automation templates, you need to have:
+
+* [GitHub Account](https://github.com/join)
+* [GitHub Repository](https://docs.github.com/en/github/getting-started-with-github/create-a-repo)
```

Comment:
To reduce friction further for users, we could suggest a repository name, e.g.:

- A [GitHub Repository](https://docs.github.com/en/github/getting-started-with-github/create-a-repo), that you can name after the integration, e.g. `dev-orbit-integration`